ssh-krb-node-executor-plugin/plugin.yaml

```diff
@@ -1,8 +1,9 @@
 name: ssh-krb-node-executor
 rundeckPluginVersion: 1.0
 author: Akos Hencz, Daniel Fernández, Philippe Ganz
-date: 29/08/2018
-version: 1.4.1
+date: 2022-08-04
```

Comment:
I'd rather to bump the version separately and keep this merge request self-contained (only the feature itself). Would that be okay? Same for other metadata changes like the release date and the project's URL.